### DISCUSSION TITLE
Culture Picker Display Problem

### DISCUSSION THREAD

**Discussion by YoumnaJ** [OrchardCMS/OrchardCore#14997] [Q&A]
Hi, in my orchard core CMS project I am using ContentCulturePicker but I have a problem that doesn't display the cultures and appears empty.
thank you for your help.

![2024-01-05_17-23-23](https://github.com/OrchardCMS/OrchardCore/assets/154245181/cbeaee94-d1c6-407c-adc7-1cc9641bc4fc)

**Comment by hishamco** [comment]
Did you add your supported cultures and enable the content culture picker feature?

**Reply by YoumnaJ** [reply]
yes, Content Culture picker feature is enabled  and I added supported cultures

**Reply by YoumnaJ** [reply]
And I added in my ContentCulturePicker template this code
![contentPickerTemlate](https://github.com/OrchardCMS/OrchardCore/assets/154245181/6a7d9497-6980-4e73-a46d-3f1277620148)

**Reply by hishamco** [reply]
Could you please re-check in TheBlogTheme, to ensure it's not a theme related issue
